test_zeroconf.py

```diff
@@ -440,6 +441,22 @@ def test_launch_and_close(self):
         rv = r.Zeroconf(interfaces=r.InterfaceChoice.Default)
         rv.close()
 
+    @unittest.skipIf(not socket.has_ipv6, 'Requires IPv6')
+    @attr('IPv6')
```

Comment:
Is the `attr` needed, since the test is already `skipIf` with the relevant arguments?

Reply:
`socket.has_ipv6` seems to check if Python supports IPv6. it passes on systems where no actual interfaces have IPv6 addresses.